tests/backup/backup_portworx_test.go

```diff
@@ -586,6 +586,35 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", func() {
 			err = CreateRestore(restoreName, backupNames[1], nil, destinationClusterName, orgID, ctx, make(map[string]string))
 			log.FailOnError(err, "%s restore failed", restoreName)
 		})
+		// PB-3962
```

Comment:
Can we add some more info here then just the ticket?

Reply:
done